docs/src/appendix/sncast/declare-deploy.md

```diff
@@ -0,0 +1,50 @@
+# `declare-deploy`
+Send subsequent declare and deploy transactions of Cairo contract to Starknet at once.
+
+> 📝 **Note**
+> This command relies on auto-estimation and does not allow specifying max fees explicitly.
+
+> 📝 **Note**
+> Only a `fee-token` can be specified. Transaction versions for both declaration and deployment are inferred from token type.
+
+## Required Common Arguments — Passed By CLI or Specified in `snfoundry.toml`
```

Comment:
Why not just `Required Arguments`?

Reply:
They are passed *before* the subcommand, e.g. `sncast --account <account> declare-deploy [...]`